falcon/testing/client.py

```diff
@@ -99,6 +99,11 @@ class Cookie:
             transmitted from the client via HTTPS.
         http_only (bool): Whether or not the cookie may only be
             included in unscripted requests from the client.
+        same_site (str): Specifies whether cookies are send in
+            cross-site requests. Possible values are 'Lax', 'Strict'
+            and 'None'. ``None`` if not specified.
+        partitioned (bool): Whether third-party cookies should be stored
+            with 2 keys.
```

Comment:
I think it would be clearer if we just said something like "Indicates if the cookie has the partitioned flag set"